### PR TITLE
parquet-cli: Patch snappy version to 1.1.8.4 for MacOS Arm64 support

### PR DESCRIPTION
**Pull request by shanielh**
- [X] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
-[X] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [X] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [X] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [X] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

### CODE REVIEW

File: Formula/parquet-cli.rb [hunk 1]
```diff
@@ -27,6 +27,13 @@ class ParquetCli < Formula
     sha256 "5caf572cb0df5ce9d6893609de82d2369b42c3c81c611847b6f921d912040118"
   end
 
+  # Patches snappy to 1.1.8.4 for MacOS arm64 support, won't be needed in >= 1.13.0
+  # See https://issues.apache.org/jira/browse/PARQUET-2025
```
Comment (SMillerDev) [inline Formula/parquet-cli.rb:31]: Why not this commit: https://github.com/apache/parquet-mr/commit/095c78fec3378189296d38fede1255b0a4d05fd4 ?

Comment (shanielh) [reply Formula/parquet-cli.rb:31]: We can do this oneC do you want me to do the change and fixup, or in a new commit?

Comment (SMillerDev) [reply Formula/parquet-cli.rb:31]: Preferably it should all be one commit.

Comment (kylemart) [reply Formula/parquet-cli.rb:31]: Any updates on this @shanielh ?

Comment (kylemart) [reply Formula/parquet-cli.rb:31]: Hey @SMillerDev , if I create a new PR that uses https://github.com/apache/parquet-mr/commit/095c78fec3378189296d38fede1255b0a4d05fd4 for the patch, would it be alright to ping you for approval? :)

Comment (shanielh) [reply Formula/parquet-cli.rb]: Hi guys, sorry for the delay, I changed the fix to use that commit.

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @shanielh  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by kylemart** [comment]
It seems this might not have fixed the issue :( Installed the patched version, but I'm still encountering the same error

**Pull request closed without merging**